contracts/token/utils/ERC721_Holder.cairo

```diff
@@ -19,16 +24,22 @@ func onERC721Received{
     return (0x150b7a02)
 end
 
-# ERC721's `safeTransferFrom` requires a means of differentiating between account and
-# non-account contracts. Currently, StarkNet does not support error handling from the
-# contract level; therefore, this ERC721 implementation requires that all contracts that
-# support safe ERC721 transfers (both accounts and non-accounts) include the `is_account` 
-# method. This method should return `0` since this contract is NOT an account.
 @view
-func is_account{
+func supportsInterface{
+        syscall_ptr : felt*,
+        pedersen_ptr : HashBuiltin*,
+        range_check_ptr
+    }(interfaceId: felt) -> (success: felt):
+    let (success) = ERC165_supports_interface('0x150b7a02')
```

Comment:
```suggestion
    let (success) = ERC165_supports_interface(interfaceId)
```

this doesn't make sense. shouldn't this function simply expose the library function?